__tests__/app.test.js

```diff
@@ -252,20 +253,37 @@ describe("DELETE:/api/comments/:comment_id", () => {
   it("204: When input an id should delete the given comment and return empty", () => {
     return request(app).delete("/api/comments/1").expect(204);
   });
+  it("400: Returns correct error when an invalid id is input", () => {
+    return request(app)
+      .delete("/api/comments/invalid")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("incorrect input");
+      });
+  });
+  it("404: Should return corrcet error when a comment is not found", () => {
+    return request(app)
+      .delete("/api/comments/100")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Comment was not found");
+      });
+  });
 });
-it("404: Should return corrcet error when a comment is not found", () => {
-  return request(app)
-    .delete("/api/comments/100")
-    .expect(404)
-    .then((response) => {
-      expect(response.body.msg).toBe("Comment was not found");
-    });
-});
-it("400: Returns correct error when an invalid id is input", () => {
-  return request(app)
-    .delete("/api/comments/invalid")
-    .expect(400)
-    .then((response) => {
-      expect(response.body.msg).toBe("incorrect input");
-    });
+describe("GET:/api/users", () => {
+  it("200: Should return an array consisting of all users containing the following properties: username, name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { allUsers } }) => {
```

Comment:
Make sure to check the length of the array to ensure you're getting the right number of responses back